add-on/chrome/modules/MozLoopAPI.jsm

```diff
@@ -186,6 +186,48 @@ const kMessageHandlers = {
   },
 
   /**
+   * Creates a layout for the remote cursor on the browser chrome,
+   * and positions it on the received coordinates.
+   *
+   * @param {Object}  message Message meant for the handler function, containing
+   *                          the following parameters in its 'data' property:
+   *                          {
+   *                            ratioX: cursor's X position (between 0-1)
+   *                            ratioY: cursor's Y position (between 0-1)
+   *                          }
+   *
+   * @param {Function} reply  Callback function, invoked with the result of the
+   *                          message handler. The result will be sent back to
+   *                          the senders' channel.
+   */
+  AddRemoteCursorOverlay: function(message, reply) {
+    let win = Services.wm.getMostRecentWindow("navigator:browser");
```

Comment:
Good point, I just made few tests pulling tabs out of the window, and certainly gave some undesired results (seeing one tab and receiving the desktop cursor from it, but sending standalone cursor to a different one). 

I agree that keeping reference to the original window is a good idea, but that change is probably better on a different bug, as there're a lot of references to `Services.wm.getMostRecentWindow("navigator:browser")` that affect other areas aside from cursor sharing.